feature-runner/lib/websocket.ts

```diff
@@ -30,7 +30,8 @@ export const createWebsocketClient = ({
 						.on('open', () => {
 							resolve()
 						})
-						.on('error', () => {
+						.on('error', (error) => {
+							debug?.(`!! Connection error: ${error.message}`)
```

Comment:
```suggestion
							debug?.(`Connection error: ${error.message}`)
```

We don't use `!!` to mark error logs.